### PR TITLE
Improve tests in SCALrecommendation

### PR DESCRIPTION
**Pull request by berland**

### CODE REVIEW

File: pyscal/scalrecommendation.py [hunk 1]
```diff
@@ -169,12 +169,6 @@ def interpolate(
                 and self.high.gasoil is not None
             )
 
-        if not do_gaswater:
-            if not do_wateroil and not do_gasoil:
-                raise ValueError(
-                    "Neither WaterOil or GasOil is complete in SCAL recommendation"
-                )
-
```
Comment (berland) [inline pyscal/scalrecommendation.py:177]: This code has proven impossible to reach.

File: pyscal/scalrecommendation.py [hunk 2]
```diff
@@ -208,11 +202,6 @@ def interpolate(
         tagstring = "\n".join(tags)
         if do_gaswater:
             interpolant = GasWater(h=h, tag=tagstring)
-            if gasparameter != parameter:
-                logger.warning(
-                    "Different interpolation parameters for Water and for "
-                    "gas in GasWater, this is maybe not what you want"
-                )
```
Comment (berland) [inline pyscal/scalrecommendation.py:215]: There is a warning a few lines above that says the same, so this one is dropped.

**Comment by berland** [comment]
The test error is solved in #292

**Review by vkip — APPROVED** [review]
LGTM

**Pull request merged**